src/nodes.coffee

```diff
@@ -3498,6 +3506,35 @@ exports.Op = class Op extends Base
   toString: (idt) ->
     super idt, @constructor.name + ' ' + @operator
 
+  ast: ->
```

Comment:
Here's another nice pattern for overriding `ast()`: do some error checking, then call out to `super()` like usual

Reply:
That is a good pattern, but why not check in the constructor? Is that not possible?